app/src/components/dialog/EditDialog.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import EditDialog from 'components/dialog/EditDialog';
+import { useFormikContext } from 'formik';
+import { TextField } from '@material-ui/core';
+import yup from 'utils/YupSchema';
+
+export interface ISampleFormikForm {
+  testField: string;
+}
+
+export const SampleFormikFormInitialValues: ISampleFormikForm = {
+  testField: ''
+};
+
+export const SampleFormikFormYupSchema = yup.object().shape({
+  testField: yup.string().max(3000, 'Cannot exceed 3000 characters').required('You must a test field for the project')
+});
+
+const SampleFormikForm = () => {
+  const formikProps = useFormikContext<ISampleFormikForm>();
+
+  const { values, handleChange, handleSubmit } = formikProps;
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <TextField
+        id="testField"
+        name="testField"
+        label="Test Field"
+        multiline
+        required={true}
+        rows={4}
+        fullWidth
+        variant="outlined"
+        value={values.testField}
+        onChange={handleChange}
+      />
+    </form>
+  );
+};
+
+describe('ErrorDialog', () => {
+  it('matches the snapshot with no error message', () => {
```

Comment:
I dont see any error message being used or shown in the editdialog component - I think it needs to be added and the i18n values used in it and then we can update this test to show an error state and the commented test can become the no-error state test